src/analyses/variable-sensitivity/abstract_environment.cpp

```diff
@@ -337,10 +337,9 @@ abstract_object_pointert abstract_environmentt::abstract_object_factory(
     type, top, bttm, e, environment, ns);
 }
 
-abstract_object_pointert abstract_environmentt::add_object_context(
-  const abstract_object_pointert &abstract_object) const
+const vsd_configt &abstract_environmentt::configuration() const
 {
-  return object_factory->wrap_with_context(abstract_object);
+  return object_factory->config();
 }
```

Comment:
Bit of a diff mismatch. `add_object_context` is redundant and was removed. `configuration` is a new member function, which we need :)